src/anniversary/cri/libadxe/sj_utl.c

```diff
@@ -0,0 +1,26 @@
+#include "common.h"
+#include <cri/private/libadxe/sj_utl.h>
+
+void SJ_SplitChunk(SJCK *ck, s32 nbyte, SJCK *ck1, SJCK *ck2) {
```

Comment:
I think we should use CRI types as much as possible
```suggestion
void SJ_SplitChunk(SJCK *ck, Sint32 nbyte, SJCK *ck1, SJCK *ck2) {
```
This would mean including `<cri/cri_xpts.h>` explicitly in this file